.github/workflows/_linux-external-build-main.yml

```diff
@@ -0,0 +1,323 @@
+name: linux-external-build
```

Comment:
Copying this from `_linux_build` is a good start but it has too many unrelated stuffs in it like s390x or scribe.  I think this needs to be cleaned up a bit

Reply:
Yeah I feel like if we're doing this copy over of this with not much changing then why do the copy at all? Is this just to support the CLI from the previous 2 commits?

Reply:
Hmm, instead of using a new reusable workflow here, what do you think about having this as a GitHub action like  `setup-nvidia` and call it inside `_linux-build.yml`?  I can see that you won't need lots of boilerplate code in `_linux-external-build-main.yml` anymore.  Inside that GitHub action, let's call it `build-external-depedency`, the input will be `build-target`, and you could just call Lumos CLI there.  This approach requires that `_linux-build.yml` be updated to have "yet" another input parameter `build-external-packages` where you could set vllm as a target and that doesn't sound too bad IMO